containers/podman-desktop
# feat: add purple dot when new content is available in dashboard (#4043)

### What does this PR do?

It adds a component that check if new content is available in the dashboard and show a purple dot accordingly.

### Screenshot/screencast of this PR

![dot_new_content](https://github.com/containers/podman-desktop/assets/49404737/1323fbac-14c4-4368-bd9f-e0c4c2551449)

### What issues does this PR fix or reference?

it closes #4043 

### How to test this PR?

1. open desktop
2. remove all podman machines so a new notification is pushed or uninstall/install an extension that adds a provider
3. you should see a dot next to the dashboard icon
4. it should be removed once you visit the dashboard
